### DISCUSSION TITLE
Is it possible to change used ckpt file in test web ui?

### DISCUSSION THREAD

**Discussion by zuencap** [TheLastBen/fast-stable-diffusion#351] [Q&A] [answered]
It would be cool to be able to use checkpoint pulldown menu in the ui. Is there a way to add ckpt files into that menu?
Do I need to stop the test and rerun with different ckpt file path?

**Comment by TheLastBen** [comment, marked as answer]
are the models in the same folder ?

**Reply by zuencap** [reply]
Thanks, I didn't notice you can specify a folder path
